go.mod

```diff
@@ -28,8 +29,8 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/google/btree v1.1.3
 	github.com/google/uuid v1.6.0
-	github.com/gopacket/gopacket v1.3.1
-	github.com/hashicorp/memberlist v0.5.2
+	github.com/gopacket/gopacket v1.3.2-0.20241202175635-b43272ae1eb8
```

Comment:
I think this is already updated in another PR https://github.com/antrea-io/antrea/pull/6972, please don't forget to fetch the latest main codes and do rebase.